ui/jstests/test-learning-resource.jsx

```diff
@@ -176,17 +176,128 @@ define(['QUnit', 'jquery', 'react', 'lodash', 'learning_resources',
     function (assert) {
       var done = assert.async();
 
+      var vocab = $.extend({}, selectedVocabulary);
+      var appendTermSelectedVocabulary = function(term) {
+        vocab.terms = vocab.terms.concat(term);
+      };
+
+      var loadedState;
+      var setLoadedState = function(loaded) {
```

Comment:
It should be a boolean, I don't think we have any conventions on parameters. There is JSDoc but I'm not sure it's necessary for functions with a limited purpose like this